docs/requirements.md

```diff
@@ -11,13 +11,27 @@ The next sections describe in detail how to set up these prerequisites.
 
 The cluster running Codacy must satisfy the following requirements:
 
--   The infrastructure hosting the cluster must be provisioned with the hardware requirements described below
+-   The infrastructure hosting the cluster must be provisioned with the hardware and networking requirements described below
 -   The orchestration platform managing the cluster must be one of:
     -   [Kubernetes](https://kubernetes.io/) **version 1.14.\*** or **1.15.\***
     -   [MicroK8s](https://microk8s.io/) **version 1.15**
 -   Tiller, the server part of [Helm](https://v2.helm.sh/) **version 2.16**, must be installed in the cluster
 -   The [NGINX Ingress controller](https://github.com/kubernetes/ingress-nginx) must be installed and correctly set up in the cluster
 
+### Cluster networking requirements
+
+The cluster must be configured to accept and establish connections on the following ports:
+
+|          | Service      | Protocol/Port | Notes                                                        |
+| -------- | ------------ | ------------- | ------------------------------------------------------------ |
+| Inbound  | SSH          | TCP/22        | Optional, to access the infrastructure remotely.             |
+| Inbound  | HTTP         | TCP/80        |                                                              |
+| Inbound  | HTTPS        | TCP/443       |                                                              |
+| Outbound | PostgreSQL   | TCP/5432      | Connection to the PostgreSQL DBMS                            |
+| Outbound | SMTP         | TCP/25        | Connection to your SMTP server                               |
+| Outbound | SMTPS        | TCP/465       | Connection to your SMTP server over TLS/SSL                  |
+| Outbound | Git provider | \*            | Connection to the ports required by your remote Git provider |
+
```

Comment:
I'm not sure if this information is enough for customers, or if we need to go into the detail of specifying which IP addresses / ranges are involved. :thinking: 

From my point of view we should have this simplified information for now and only add more detail if customers ask for it.